.github/workflows/commit-messages-check.yml

```diff
@@ -0,0 +1,14 @@
+name: "[Check]: Commit messages"
```

Comment:
Wouldn't make sense to move also fixup check to this flow? It's basically same thing, just validation of commit msg.